### PR TITLE
Chrome stable for travis-ci

### PR DESCRIPTION
**Pull request by wardpeet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,3 +51,5 @@ after_success:
   - ./lighthouse-core/scripts/generate-combined-coverage.sh
 after_failure:
   - grep 'No screenshots' perf.json && travis-artifacts upload --path perf-0.trace.json
+addons:
+  chrome: stable
```
Comment (stramel) [inline .travis.yml]: Just curious if this is beneficial over

```yml
addons:
  apt:
    sources:
      - google-chrome
    packages:
      - google-chrome-stable
```

**Comment by wardpeet** [comment]
When we use the latest chromium build on trusty we get
```
/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by /home/travis/builds/GoogleChrome/lighthouse/chrome-linux/chrome)
/home/travis/builds/GoogleChrome/lighthouse/chrome-linux/chrome: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.19' not found (required by /home/travis/builds/GoogleChrome/lighthouse/chrome-linux/chrome)
```

File: .travis.yml (hunk 1 above)
Comment (wardpeet) [inline .travis.yml]: It's probably an abstraction travis-ci is making so it doesn't matter what distro you're on
https://docs.travis-ci.com/user/chrome

**Comment by wardpeet** [comment]
DON'T MERGE JUST YET ;)

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ cache:
   yarn: true
   directories:
     - node_modules
-    - chrome-linux
+    #- chrome-linux
```
Comment (paulirish) [inline .travis.yml]: lets remove.

File: .travis.yml [hunk 3]
```diff
@@ -30,7 +30,8 @@ before_script:
   - export DISPLAY=:99.0
   - export LIGHTHOUSE_CHROMIUM_PATH="$(pwd)/chrome-linux/chrome"
   - sh -e /etc/init.d/xvfb start
-  - ./lighthouse-core/scripts/download-chrome.sh
+  - sleep 3 # wait for xvfb to boot
+  #- ./lighthouse-core/scripts/download-chrome.sh
```
Comment (paulirish) [inline .travis.yml]: remove

**Pull request merged**